SpatialGDK/Source/SpatialGDK/Private/EngineClasses/GridBasedLBStrategy.cpp

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "EngineClasses/GridBasedLBStrategy.h"
+
+#include "EngineClasses/SpatialNetDriver.h"
+#include "Utils/SpatialActorUtils.h"
+
+UGridBasedLBStrategy::UGridBasedLBStrategy()
+	: Super()
+	, Rows(1)
+	, Cols(1)
+	, WorldWidth(10000.f)
+	, WorldHeight(10000.f)
+{
+}
+
+void UGridBasedLBStrategy::Init(const USpatialNetDriver* InNetDriver)
+{
+	Super::Init(InNetDriver);
+
+	for (uint32 i = 1; i <= Rows * Cols; i++)
+	{
+		VirtualWorkerIds.Add(i);
+	}
+
+	const float WorldWidthMin = -(WorldWidth / 2.f);
+	const float WorldHeightMin = -(WorldHeight / 2.f);
+
+	const float ColumnWidth = WorldWidth / Cols;
+	const float RowHeight = WorldHeight / Rows;
+
+	float XMin = WorldWidthMin;
+	float YMin = WorldHeightMin;
+	float XMax, YMax;
+
+	for (uint32 Col = 0; Col < Cols; ++Col)
+	{
+		XMax = XMin + ColumnWidth;
+
+		for (uint32 Row = 0; Row < Rows; ++Row)
+		{
+			YMax = YMin + RowHeight;
+
+			FVector2D Min(XMin, YMin);
+			FVector2D Max(XMax, YMax);
+			FBox2D Cell(Min, Max);
+			WorkerCells.Add(MoveTemp(Cell));
```

Comment:
For POD move and copy are the same thing.

Reply:
Done